packages/experience/src/hooks/use-password-policy-checker.ts

```diff
@@ -0,0 +1,32 @@
+import { useCallback } from 'react';
+
+import usePasswordErrorMessage from './use-password-error-message';
+import { usePasswordPolicy } from './use-sie';
+
+type PasswordPolicyCheckProps = {
```

Comment:
nit: I notice that in our code base, we always use `...Props` for react components, and use `...Options` for hooks. Should we stay consistent with the previous code?